External/Plugins/CodeRefactor/Provider/ICommandFactory.cs

```diff
@@ -122,23 +122,23 @@ public interface ICommandFactory
         /// Outputs found results.
         /// Uses the current text location as the declaration target.
         /// </summary>
-        Command CreateRenameCommand();
+        Command CreateRenameCommandAndExecute();
```

Comment:
Yes, it starts execution right after the creation. This appeared after adding "Inline renaming". I'll fix it, but it takes time